<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemLiquidPipeWashWidgetsClass</class>
 <widget class="QWidget" name="SystemLiquidPipeWashWidgetsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1660</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SystemLiquidPipeWashWidgets</string>
  </property>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>330</y>
     <width>571</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>请选择泵</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="cb_all">
       <property name="text">
        <string>全选</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>30</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>30</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>30</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>30</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>1181</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="chk_1">
       <property name="text">
        <string>1号泵</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_2">
       <property name="text">
        <string>2号泵</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_3">
       <property name="text">
        <string>3号泵</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_4">
       <property name="text">
        <string>4号泵</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_5">
       <property name="text">
        <string>5号泵</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_6">
       <property name="text">
        <string>6号泵</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_7">
       <property name="text">
        <string>7号泵</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_8">
       <property name="text">
        <string>8号泵</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chk_9">
       <property name="text">
        <string>9号泵</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>167</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>160</y>
     <width>301</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>25</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>系统液管路清洗</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>————————————————————————————</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>清洗液量：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>ml</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
